include/mbedtls/config.h

```diff
@@ -2254,6 +2254,19 @@
  */
 #define MBEDTLS_ECP_C
 
+/**
+ * \def MBEDTLS_USE_UECC
```

Comment:
A bit of information that you couldn't guess, and that I'm afraid is not really documented anywhere: every time you add a new flag in `config.h` you need to run `scripts/generate_features.pl` and `scripts/generate_query_config.pl` afterwards to update files that are generated from `config.h`. Otherwise `tests/scripts/check-generated-files.sh` will fail in the CI.

(Btw, there is a suggested git pre-push script in `tests/git-scripts` that you can use if you want, though for that particular PR many of the scripts it runs are going to be broken and require fixing, so if you want to set up that git hook, you might want to wait until after you're done fixing those scripts for the new code.)